src/Moryx.Runtime.Kernel/Modules/Components/ModuleStarter.cs

```diff
@@ -93,7 +93,7 @@ private void ExecuteModuleStart(object moduleObj)
             }
             catch(Exception ex)
             {
-                _logger.LogException(LogLevel.Error, ex, "Failed to start module {0}", module.Name);
+                _logger.LogError(ex, "Failed to start module {0}", module.Name);
```

Comment:
Maybe $ string interpolation could be more readable

Reply:
Using string interpoltion in logging is discouraged for performance and stability. 

It formats the string `before` passing to the logger, that might not even write it depending on current log level. It could also cause exceptions that are otherwise handled by the logger.